integration/qbft/scenarios/regular.go

```diff
@@ -57,115 +61,160 @@ func Regular(role spectypes.BeaconRole) *IntegrationTest {
-				specqbft.FirstRound: {
-					&specqbft.SignedMessage{
-						Message: &specqbft.Message{
-							Data: commitData,
-						},
-					},
-				},
-			}},
-			RoundChangeContainer: &specqbft.MsgContainer{Msgs: map[specqbft.Round][]*specqbft.SignedMessage{}},
-		},
-		DecidedMessage: &specqbft.SignedMessage{
-			Message: &specqbft.Message{
+				Data:       prepareData,
+			})
+			if !isMessageExistInRound(expectedMsg, actual.State.PrepareContainer.Msgs[specqbft.FirstRound]) {
+				return fmt.Errorf("prepare message %+v wasn't found at actual.State.PrepareContainer.Msgs[specqbft.FirstRound]", expectedMsg)
+			}
+
+			expectedMsg = spectestingutils.SignQBFTMsg(spectestingutils.Testing4SharesSet().Shares[operatorIDIterator], operatorIDIterator, &specqbft.Message{
 				MsgType:    specqbft.CommitMsgType,
 				Height:     specqbft.FirstHeight,
 				Round:      specqbft.FirstRound,
 				Identifier: identifier[:],
-				Data:       spectestingutils.PrepareDataBytes(consensusData),
+				Data:       commitData,
+			})
+			if isMessageExistInRound(expectedMsg, actual.State.CommitContainer.Msgs[specqbft.FirstRound]) {
+				foundedMsgsCounter++
+			}
+		}
+
+		if !actual.State.Share.HasQuorum(foundedMsgsCounter) {
+			return fmt.Errorf("wasn't found enough commit messages at actual.State.CommitContainer.Msgs[specqbft.FirstRound], expected at least %d, actual = %d", actual.State.Share.Quorum, foundedMsgsCounter)
+		}
+
+		actual.State.ProposeContainer = nil
+		actual.State.PrepareContainer = nil
+		actual.State.CommitContainer = nil
+
+		expectedStoredInstance := &protocolstorage.StoredInstance{
+			State: &specqbft.State{
+				Share:             testingShare(spectestingutils.Testing4SharesSet(), operatorID),
+				ID:                identifier[:],
+				Round:             specqbft.FirstRound,
+				Height:            specqbft.FirstHeight,
+				LastPreparedRound: specqbft.FirstRound,
+				LastPreparedValue: consensusData,
+				ProposalAcceptedForCurrentRound: spectestingutils.SignQBFTMsg(spectestingutils.Testing4SharesSet().Shares[1], 1, &specqbft.Message{
+					MsgType:    specqbft.ProposalMsgType,
+					Height:     specqbft.FirstHeight,
+					Round:      specqbft.FirstRound,
+					Identifier: identifier[:],
+					Data:       proposalData,
+				}),
+				Decided:      true,
+				DecidedValue: consensusData,
+
+				RoundChangeContainer: &specqbft.MsgContainer{Msgs: map[specqbft.Round][]*specqbft.SignedMessage{}},
 			},
-		},
+			DecidedMessage: &specqbft.SignedMessage{
+				Message: &specqbft.Message{
+					MsgType:    specqbft.CommitMsgType,
+					Height:     specqbft.FirstHeight,
+					Round:      specqbft.FirstRound,
+					Identifier: identifier[:],
+					Data:       spectestingutils.PrepareDataBytes(consensusData),
+				},
+			},
+		}
+
+		expectedStateRoot, err := expectedStoredInstance.State.GetRoot()
+		if err != nil {
+			return fmt.Errorf("error during geting root from expected state: %w", err)
```

Comment:
@egorka-sereda 

Errors are usually chained, so if we have a similar message somewhere inside or outside, we will get something like `error during A: error during B: actual error`, which will be very long. Can we make it shorter? Something like this:
```suggestion
			return fmt.Errorf("get expected state root: %w", err)
```